### PR TITLE
Add 1inch new router address to dex.trades (V1)

### PR DESCRIPTION
**Pull request by fatayri**
Brief comments on the purpose of your changes:

Hey! We recently released a new version of our router, V5. Since a lot of dashboards are still actively using dex.trades on V1, I added the trades of the new router to this table.

### CODE REVIEW

File: deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql [hunk 1]
```diff
@@ -283,6 +289,18 @@ WITH rows AS (
             where call_success
                 and call_block_time >= start_ts
                 and call_block_time < end_ts
+            -- union all
+            -- select "output_returnAmount", "amount", "srcToken", "pools", "call_tx_hash", "call_trace_address", "call_block_time", "contract_address"
```
Comment (jeff-dude) [inline deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql:293]: the ```pools``` column in both of these v5 decoded tables are numeric data type, while prior tables in the union are bytea, causing failures and unable to load. i've commented these out and continued loading

Comment (gentrexha) [reply deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql:293]: Is this the reason why 1inch isn't in `dex.trades` currently?

Comment (jeff-dude) [reply deprecated-dune-v1-abstractions/ethereum/dex/trades/insert_1inch.sql:293]: yes, it's reprocessing history with changes here (it takes a long time)

**Review by jeff-dude — APPROVED** [review]

**Comment by augustog** [comment]
Team (@fatayri) , could any of the changes made here be overstating token amounts? We're seeing a significant increase in 1inch volumes today.

https://dune.com/queries/1677451

cc @jeff-dude

**Comment by jeff-dude** [comment]
> https://dune.com/queries/1677451

query is private

**Comment by bh2smith** [comment]
Hey @jeff-dude -- Do you know what is the matter with this? There are no records since November 30th for 1inch.

**Comment by jeff-dude** [comment]
please refer to #2117

**Pull request merged**